src/server/v2.0/handler/auditlog.go

```diff
@@ -26,28 +25,9 @@ func (a *auditlogAPI) ListAuditLogs(ctx context.Context, params auditlog.ListAud
 	// if !a.HasPermission(ctx, rbac.ActionList, rbac.ResourceLog) {
```

Comment:
Will we do the permission check in another PR?

Reply:
yes  to resolve ToDo items